docs/tutorials/get-started/confetti-bot.md

```diff
@@ -17,7 +17,7 @@ date: "2023-05-29"
 cost: 145
 ---
 
-{{<gif webm_src="/tutorials/confetti-bot/confetti-bot.webm" mp4_src="/tutorials/confetti-bot/confetti-bot.mp4" alt="GIF of a red button being pressed and the cannon spraying confetti" class="alignright" max-width="250px">}}
+{{<gif webm_src="/tutorials/confetti-bot/confetti-bot-explodes.webm" mp4_src="/tutorials/confetti-bot/confetti-bot-explodes.mp4" alt="GIF of red button being pressed and cannon of confetti bot spraying confetti" class="alignright" max-width="275px">}}
```

Comment:
let's also change line 7 and 8 to use the confetti-bot-explodes files and for line 10 replace that gif with the new one

Reply:
@npentrel done!!